sdk/containerregistry/Azure.Containers.ContainerRegistry/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
+## 1.2.0 (2023-11-21)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Enable mocking for extension methods, refer this [document](https://aka.ms/azsdk/net/mocking) for more details.
```

Comment:
What changed in the library that makes this true?

Reply:
This feature is tracked by this [issue[(https://github.com/Azure/autorest.csharp/issues/3493) which has been done.

Reply:
Thank you for pointing this out, changed by mistake, reverted.